Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1520,6 +1522,11 @@ - (NSString *)cacheKeyForURL:(NSURL *)url processorKey:(NSString *)processorKey
 
 - (void)objectForKey:(NSString *)key options:(PINRemoteImageManagerDownloadOptions)options completion:(void (^)(BOOL found, BOOL valid, PINImage *image, id alternativeRepresentation))completion
 {
+    if (options & PINRemoteImageManagerDownloadOptionsIgnoreCache) {
+        completion(NO, NO, nil, nil);
```

Comment:
I think the second param should be YES? Otherwise PINRemoteImage will report errors about invalid items in the cache.